test/src/integration/templates/enableRepo.test.js

```diff
@@ -12,116 +12,100 @@ const chai = require('chai');
-            res.should.have.status(207);
-            res.body[0].status.should.equal(200);
-        });
-        it(`that repo appears as enabled in list of template repos`, async function() {
-            const res = await getTemplateRepos();
-            res.should.have.status(200);
-            res.body.should.deep.include({
-                ...repoToTest,
-                enabled: true,
+describe('Batch enabling repositories', function() {
+    const tests = {
+        '1 repo': {
+            testRepos: [{ ...sampleRepos.codewind }],
+        },
+        'multiple repos': {
+            testRepos: [
+                { ...sampleRepos.codewind },
+                { ...sampleRepos.appsody },
+            ],
+        },
+    };
+
+    for (const [testName, test] of Object.entries(tests)) {
+        describe(testName, function() { // eslint-disable-line
+            let originalTemplateRepos;
+            const { testRepos } = test;
+            let templatesFromTestRepos;
+            before(async() => {
+                const res = await getTemplateRepos();
+                originalTemplateRepos = res.body;
+
+                await resetTemplateReposTo(testRepos);
+
+                const res2 = await getTemplates();
+                templatesFromTestRepos = res2.body;
             });
-        });
-        it(`batch disabling that template repo returns 207 and sub-status 200`, async function() {
-            const res = await disableTemplateRepos([repoToTest.url]);
-            res.should.have.status(207);
-            res.body[0].status.should.equal(200);
-        });
-        it(`that repo appears as disabled in list of template repos`, async function() {
-            const res = await getTemplateRepos();
-            res.should.have.status(200);
-            res.body.should.deep.include({
-                ...repoToTest,
-                enabled: false,
+            after(async() => {
+                await resetTemplateReposTo(originalTemplateRepos);
             });
-        });
-        it(`templates from the disabled repo do not appear in list of enabled templates`, async function() {
-            const res = await getTemplates({ showEnabledOnly: true });
-            res.should.have.status(204);
-            // TODO: switch over to the below when we have 2 repos (so that 1 repo can still be enabled, so we return 200 rather than 204 No Content)
-            // res.should.have.status(200)
-            // res.body.should.not.include.deep.members(defaultTemplates);
-        });
-    });
-    describe('when enabling multiple template repositories', function() {
-        let originalTemplateRepos;
-        let reposToTest;
-        before(async() => {
-            const res = await getTemplateRepos();
-            originalTemplateRepos = res.body;
+            it(`batch DISabling ${testRepos.length} repos returns 207 and sub-status 200 for each subrequest`, async function() {
```

Comment:
Doesn't affect functionality and personal preference I guess,  the DISabling / DISabled / ENabled capitalisation throughout just looks a bit odd to me.